app/components/agenda/agendaitem/decision/digital.js

```diff
@@ -312,12 +312,24 @@ export default class AgendaAgendaitemDecisionDigitalComponent extends Component
     const documents = this.pieces;
     const agendaActivity = await this.args.agendaitem.agendaActivity;
     const subcase = await agendaActivity?.subcase;
-    const newBetreftContent = await generateBetreft(shortTitle,
-      title,
-      this.args.agendaitem.isApproval,
-      documents,
-      subcase?.subcaseName
-    );
+    let newBetreftContent;
+    if (subcase?.isBekrachtiging) {
```

Comment:
`subcase?.isBekrachtiging` (sync getter) needs `await subcase?.type` to resolve.